_data/settings/utilities.yml

```diff
@@ -60,6 +60,13 @@ contents:
         value: "false"
         type: boolean
 
+  - name: Custom breakpoints
+    description: |
+      Set additional custom mobile-first breakpoints to be output as utilities alongside the enabled system breakpoints. Values must be passed in a scss map as px.
```

Comment:
**suggestion**: Small tweaks to align with description for `$theme-utility-breakpoints` and to improve clarity. 
```suggestion
      Set additional custom mobile-first breakpoints to be output for the utilities. Values must be passed in with `px` values inside a Sass map.
```